src/main/scripts/src/app/collection/config/attribute-list/attribute-list.component.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../../shared/_lumeer-variables";
+
+:host {
```

Comment:
Why? Angular take care of keeping the separate styles for different components.

Reply:
It didn't when I used browser for navigating between pages and adding the `:host` fixed the issue.